components/Composition/FAnnouncementCard.jsx

```diff
@@ -11,86 +10,116 @@ import icons from 'themes/icons';
 import colors from 'themes/colors';
 import { FCard } from 'components/Composition/FCard';
 import placements from 'themes/placements';
-import images from 'constants/images';
+import { useNavigation } from '@react-navigation/native';
+import PropTypes from 'prop-types';
 
-export const FAnnouncementCard = ({ width, data, height }) => (
-  <TouchableWithoutFeedback>
-    <View style={{
-      padding: sizes.PADDING_5,
-    }}
-    >
-      <FCard
-        width={width}
-        paddingHorizontal={sizes.PADDING_10}
-        paddingVertical={sizes.PADDING_10}
+export const FAnnouncementCard = ({
+  width, data, height, link, titleStyle,
```

Comment:
propsa link nie potrzebujemy, pisałam wczoraj że teraz jak już wiemy jak to działa to jedynie potrzebujemy id ogłoszenia żeby nas przeniosło tam gdzie chcemy :D

Reply:
titleStyle też do wywalenia